test/chaat/db/user_test.clj

```diff
@@ -0,0 +1,71 @@
+(ns chaat.db.user-test
+  (:require [chaat.fixture :as fixture]
+            [clojure.test :refer :all]
+            [chaat.db.user :as db.user]
+            [chaat.model.user :as model.user]
+            [crypto.password.bcrypt :as bcrypt]
+            [java-time.api :as jt]))
+
+(use-fixtures :each fixture/test-fixture)
```

Comment:
suggestion: for many/most of the tests, you actually don't need all the setup. for eg. here, you just need the db connection established and http server is not necessary. 
If you separate out fixtures so that one fixture has one responsibility, we can selectively choose what to use in the setup of different tests.

Reply:
This can be addressed in a separate PR, but its good to think abt this